testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -1183,6 +1183,50 @@ QUnit.module('Templates', () => {
     }
 });
 
+QUnit.module('Drop Down Popup', {
```

Comment:
Why do we need separate high-level module for this tests? We already have `popup integration` module. Let's move this module there, name it `ios tests` and move tests' common code to beforeEach

Reply:
https://github.com/DevExpress/DevExtreme/pull/22834/commits/f94bb40f05bacbe31caa6768f9e639e701d615da